app/js/lib/foxbox/foxbox.js

```diff
@@ -355,7 +355,7 @@ export default class Foxbox extends Service {
     return this[p.net].fetchJSON(`${this[p.net].origin}/services/${id}/state`)
       .then(res => {
         if (!res) {
-          throw new Error(`The action couldn't be performed.`);
+          throw new Error('The action couldn\'t be performed.');
```

Comment:
Although it seems we can use ```quotes: ["error", "single", "backtick"]``` to enable backticks for regular strings, I have a mixed feelings about using backticks for ordinary strings, maybe it's just me